animation_url/src/routes/[slug]/+page.ts

```diff
@@ -7,14 +8,31 @@ const isDev =
 
 const chain = isDev ? "polygon-mumbai" : "ethereum";
 const deployment = deployments[chain];
+
 const ipfsGatewayUri = isDev
   ? "https://nftstorage.link/ipfs/"
   : "https://tableland.mypinata.cloud/ipfs/";
 
+// NOTE: zdk docs say that only mainnet is supported currently. We could be
+//       supporting testnet here, but for the time being testnet localnet
+//       collections will show the pilot from mainnet.
+const networkInfo = {
+  network: ZDKNetwork.Ethereum,
+  chain: ZDKChain.Mainnet,
```

Comment:
The [zora labs zdk docs](https://www.npmjs.com/package/@zoralabs/zdk) mention that they only support mainnet ethereum.  This means only mainnet ethereum NFT pilots can be viewed.